src/data/faq.json

```diff
@@ -30,16 +30,86 @@
+                                    "Yes, the source code of the CWA will remain accessible in the GitHub repositories. The duration of the source code's reachability will be based on GitHub's processes and policies.",
+                                    "Until May 31, 2023, the source code will be maintained as part of CWA development.",
+                                    "For handling bug reports and bug fixes in GitHub, see also  FAQ: <a href='#ramp_down_reports' target='_blank'>What if the app has malfunctions/has errors, will they not be fixed? Can I still report those bugs?</a>"
+                                ]
+                            },
+                            {
+                                "title": "Until when is the CWA mailbox read and processed?",
+                                "anchor": "ramp_down_mailbox",
+                                "active": false,
+                                "textblock": [
+                                    "Until May 31, 2023 there will be community support, in which incoming messages will be read and answered if applicable."
+                                ]
+                            },
+                            {
+                                "title": "What happens with my personal data in the emails?",
+                                "anchor": "ramp_down_mailbox_data",
+                                "active": false,
+                                "textblock": [
+                                    "An automatic deletion is set up for the CWA mailbox: All emails that are now in our CWA mailbox or still reach that mailbox will be deleted irreversibly without replacement, after 6 months at the latest, to ensure the protection of your data."
+                                ]
+                            },
+                            {
+                                "title": "What happens to the CWA's website at www.coronawarn.app?",
+                                "anchor": "ramp_down_website",
+                                "active": false,
+                                "textblock": [
+                                    "The <a href='../../' target='_blank'>Corona-Warn-App website</a> will continue to be reachable. Among other things, the Science Blog will be updated with new articles."
+                                ]
+                            },
+                            {
+                                "title": "What happens to the EFGS for cross-country risk determination? How long will this service continue to run?",
+                                "anchor": "ramp_down_efgs",
+                                "active": false,
+                                "textblock": [
+                                    "The participating member states have agreed to discontinue the EFGS (European Federation Gateway Service) in February 2023. This means all countries, including Germany, have been detached from the EFGS."

```

Comment:
"have agreed" => "agreed"

since February is now in the past.

Reply:
Updated in https://github.com/corona-warn-app/cwa-website/pull/3435/commits/19e4e13371b4215a468ab318569a73ff429312d4, thanks for the correction!